src/psyclone/tests/psyir/nodes/intrinsic_call_test.py

```diff
@@ -301,3 +321,58 @@ def test_intrinsiccall_create_errors():
                              [aref, ("stat", sym)])
     assert ("The optional argument 'stat' to intrinsic 'ALLOCATE' must be "
             "of type 'Reference' but got 'DataSymbol'" in str(err.value))
+
+
+def test_create_positional_arguments_with_names():
+    ''' Test the create method when given named positional arguments.'''
+    sym = DataSymbol("my_array",
+                     ArrayType(INTEGER_TYPE, [ArrayType.Extent.DEFERRED]))
+    aref = ArrayReference.create(sym, [Literal("20", INTEGER_TYPE)])
+    bref = ArrayReference.create(sym, [Literal("20", INTEGER_TYPE)])
+
+    # All of these are valid
```

Comment:
In the following please could you extend slightly just to check that `create` has produced what you expect.

Reply:
Done